packages/dd-trace/src/profiling/profiler.js

```diff
@@ -15,19 +13,16 @@ class Profiler extends EventEmitter {
     if (!config.enabled) return
 
     this._logger = config.logger
-
-    if (!semver.satisfies(process.version, '>=10.12')) {
```

Comment:
Does `pprof` support Node 8.0.0?

Reply:
_Sort of_. It seems it actually _builds_, it just fails if you load the built-in source mapper as it relies on a dependency which uses new language syntax only supported in 10+.

Reply:
Looks like there are some major issues in older versions: https://github.com/nodejs/node/issues/23070. Not sure if that should be considered a Node problem even though it's a foot gun.

Reply:
Yeah, I put the restriction back already.